### PR TITLE
⬆️ color@3.1.2

### PR DESCRIPTION
**Pull request by atom-build**
Bumps color from 0.7.3 to 3.1.2

### CODE REVIEW

**Review by deshnawysameh — APPROVED** [review]

**Review by deshnawysameh — APPROVED** [review]

**Review by deshnawysameh — APPROVED** [review]

**Review by deshnawysameh — APPROVED** [review]

**Review by deshnawysameh — APPROVED** [review]

**Review by deshnawysameh — APPROVED** [review]
Excellent works

**Review by deshnawysameh — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -2156,19 +2153,12 @@
       }
     },
     "color": {
-      "version": "0.7.3",
-      "resolved": "https://registry.npmjs.org/color/-/color-0.7.3.tgz",
-      "integrity": "sha1-qzrkvGy4z62110nEDzSuoIgQT4k=",
+      "version": "3.1.2",
+      "resolved": "https://registry.npmjs.org/color/-/color-3.1.2.tgz",
+      "integrity": "sha512-vXTJhHebByxZn3lDvDJYw4lR5+uB3vuoHsuYA5AKuxRVn5wzzIfQKGLBmgdVRHKTJYeK5rvJcHnrd0Li49CFpg==",
```
Comment (jos912) [inline package-lock.json:2158]: Open api

File: script/lib/update-dependency/util.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ module.exports = {
   fetchOutdatedDependencies: async function() {
     return [
       ...(await checkAPM(packageJSON)),
-      // ...(await checkNPM(repositoryRootPath))
+      ...(await checkNPM(repositoryRootPath))
     ];
   },
   updatePackageJson: async function({
```
Comment (jos912) [inline script/lib/update-dependency/util.js:18]: joshmisty JPEG gift diff jpg png log pdf mov

**Review by jos912 — CHANGES_REQUESTED** [review]

**Review by deshnawysameh — APPROVED** [review]
Goooood

**Review by deshnawysameh — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (deshnawysameh) [reply package-lock.json:2158]: Nice

**Pull request closed without merging**